react/features/mobile/external-api/middleware.js

```diff
@@ -429,3 +437,30 @@ function escapeBackslashes(value: string, alsoOnIos?: boolean) {
         ? value.replace(/\\/g, '\\\\')
         : value;
 }
+
+function getRemoteDescriptions(store) {
+    const connection = store.getState()['features/base/connection'].connection;
```

Comment:
Please use a simpler and safer form here:
const { connection} = store.getState()['features/base/connection'];
It also handles when the state is undefined.